Modules/_asynciomodule.c

```diff
@@ -3947,9 +3968,12 @@ static int
 module_exec(PyObject *mod)
 {
     asyncio_state *state = get_asyncio_state(mod);
+
+    PyMutex_Lock(&state->mutex);
```

Comment:
No locking in `module_exec`. This is called before the module is visible to other threads.